Microsoft.Toolkit.Graph.Controls/Providers/GraphExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Graph.Providers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public static class GraphExtensions
+    {
+        /// <summary>
+        /// Gets the Graph Config property value.
+        /// </summary>
+        /// <param name="target"></param>
+        /// <returns></returns>
+        public static IGraphConfig GetGraphConfig(DependencyObject target)
+        {
+            return (IGraphConfig)target.GetValue(GraphConfigProperty);
+        }
+
+        /// <summary>
+        /// Sets the GraphConfig property value.
+        /// </summary>
+        /// <param name="target"></param>
+        /// <param name="value"></param>
+        public static void SetGraphConfig(DependencyObject target, IGraphConfig value)
```

Comment:
Wonder if we want to scope the Attached Property Get/Set to be a `Page` type instead of a `DependencyObject` to help guide a single usage of one?

Reply:
The guidance would be nice, but a bit superficial. The controls should gracefully respond to the change in GlobalProvider, regardless of when/where it is set.

Reply:
Agree, the controls should respond whenever it's set. This is just about where we can attach the `<providers:Graph.Config>` helper itself within the context of the XAML tree, like:

```xml
<Page ...>
   <providers:Graph.Config>
       <providers:MockConfig/>
   </providers:Graph.Config>
```

By having the `target` be a `DependencyObject` it can go anywhere deep within a Page or included multiple times easily. I was just suggesting we change the Get/Set to be like `SetGraphConfig(Page target, ...)` so that they can at most include it once per page to help minimize potential duplication or it getting buried and lost within the page itself...

As they should really just have one single config setup per app, eh?

Reply:
Excellent points, I agree. I'll switch it to Page.